google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/st/ITBigQueryStorageLongRunningWriteTest.java

```diff
@@ -0,0 +1,307 @@
+        object.put(
+            "test_numerics3",
+            new JSONArray(
+                new String[] {
+                  "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15",
+                  "16", "17", "18", "19", "20", "21", "22", "23", "24", "25", "26", "27", "28",
+                  "29", "30", "31", "32", "33", "34", "35", "36", "37", "38", "39", "40", "41",
+                  "42", "43", "44", "45", "46", "47", "48", "49", "50", "51", "52", "53", "54",
+                  "55", "56", "57", "58", "59", "60", "61", "62", "63", "64", "65", "66", "67",
+                  "68", "69", "70", "71", "72", "73", "74", "75", "76", "77", "78", "79", "80",
+                  "81", "82", "83", "84", "85", "86", "87", "88", "89", "90", "91", "92", "93",
+                  "94", "95", "96", "97", "98", "99", "100"
+                }));
+        object.put("test_datetime", String.valueOf(LocalDateTime.now()));
+        object.put(
+            "test_bools",
+            new JSONArray(
+                new boolean[] {
+                  false, true, false, true, false, true, false, true, false, true, false, true,
+                  false, true, false, true, false, true, true, false, true, false, true, false,
+                  true, false, true, false, true, false, true, true, false, true, false, true,
+                  false, true, false, true, false, true, false, true, true, false, true, false,
+                  true, false, true, false, true, false, true, false, true, true, false, true,
+                  false, true, false, true, false, true, false, true, false, true, true, false,
+                  true, false, true, false, true, false, true, false, true, false, true, true,
+                  false, true, false, true, false, true, false, true, false, true, false, true,
+                  true, false, true, false, true, false, true, false, true, false, true, false,
+                  true, true, false, true, false, true, false, true, false, true, false, true,
+                  false, true,
+                }));
+        JSONObject sub = new JSONObject();
+        sub.put("sub_bool", true);
+        sub.put("sub_int", 12);
+        sub.put("sub_string", "Test Test Test");
+        object.put("test_subs", new JSONArray(new JSONObject[] {sub, sub, sub, sub, sub, sub}));
+        break;
+      default:
+        break;
+    }
+    return object;
+  }
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    parentProjectId = String.format("projects/%s", ServiceOptions.getDefaultProjectId());
+
+    client = BigQueryWriteClient.create();
+    RemoteBigQueryHelper bigqueryHelper = RemoteBigQueryHelper.create();
+    bigquery = bigqueryHelper.getOptions().getService();
+    dataset = RemoteBigQueryHelper.generateDatasetName();
+    DatasetInfo datasetInfo =
+        DatasetInfo.newBuilder(/* datasetId = */ dataset).setDescription(DESCRIPTION).build();
+    bigquery.create(datasetInfo);
+  }
```

Comment:
Should we log the dataset name here in case something goes very wrong (e.g. test crashes) and we need to investigate?

Reply:
Yeah that makes sense, I'll log the dataset name in case something goes wrong.